pkg/cover/html.go

```diff
@@ -185,6 +185,9 @@ func (rg *ReportGenerator) DoRawCoverFiles(w http.ResponseWriter, progs []Prog,
 			log.Logf(0, "DoRawCoverFiles: pc (%v) != frame.PC (%v) \n", pc, frame.PC)
 		}
 		offset := frame.PC - frame.Module.Addr
+		if pc != frame.PC {
+			log.Logf(0, "pc (0x%x) != frame.PC (0x%x) for module %v, offset=0x%x\n", pc, frame.PC, frame.Module.Name, offset)
```

Comment:
This log happens only when access /rawcoverfiles, it can be used as a sanity check if the kcov build has all things set, for example, if modules loaded into fixed address, if kernel module symbols are loaded with an offset, if modules has been stripped so that it cannot get frame.PC.
There 3 are the cases I can think of on especially on Android 12 target.

We use coverage a lot but for others, it's fine even if pc != frame.PC.